app/editor/src/features/admin/evening-overview/overview-section/OverviewSection.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { FaEyeSlash, FaSpinner } from 'react-icons/fa';
+import { MdAdd, MdClear, MdEdit, MdSave } from 'react-icons/md';
+import { useLookupOptions } from 'store/hooks';
+import { useEveningOverviews } from 'store/hooks/admin/useEveningOverviews';
+import { Button, ButtonVariant, Row, Show } from 'tno-core';
+
+import {
+  EveningOverviewItemType,
+  IEveningOverviewItem,
+  IEveningOverviewSection,
+} from '../interfaces';
+import { OverviewGrid } from '../overview-grid';
+import { EditBroadcastDetails } from './EditBroadcastDetails';
+import * as styled from './styled';
+
+export interface IOverviewSectionProps {
+  currentSection?: IEveningOverviewSection;
+  setSections: React.Dispatch<React.SetStateAction<IEveningOverviewSection[]>>;
+}
+
+/** This section includes the overview grid, as well as the broadcast details. */
+export const OverviewSection: React.FC<IOverviewSectionProps> = ({
+  currentSection,
+  setSections,
+}) => {
+  const [items, setItems] = React.useState<IEveningOverviewItem[]>([]);
+  const [editable, setEditable] = React.useState<boolean>(false);
+  const [section, setSection] = React.useState<IEveningOverviewSection>();
+  const [saving, setSaving] = React.useState<boolean>(false);
+
+  const [, api] = useEveningOverviews();
```

Comment:
Keep all your use* statements at the top of the component. It's part of our standard to make it easy to know what a component is dependent upon.  

`useState(...)` are included next for consistency, as they are not dependencies but state.